src/utils/errors.js

```diff
@@ -0,0 +1,54 @@
+import { app, dialog, shell } from 'electron'
+
+const issueTemplate = (e) => `Please describe what you were doing when this error happened.
```

Comment:
This file is nice and tidy. Much clear now it's split out from the createDaemon logic.